source/components/organisms/Step/Step.tsx

```diff
@@ -225,6 +225,10 @@ function Step({
     });
   };
 
+  const showCloseFormButton = !actions.some(
+    ({ type = "" }) => type === "close"
```

Comment:
Is a default needed for type? (`type = ""`)

Reply:
That's only for removing the typescript warning since none of the props are typed in this file.